src/App.js

```diff
@@ -1,16 +1,133 @@
 import React from 'react';
+import moment from 'moment';
+
+import TodoList from './components/TodoComponents/TodoList';
+import TodoForm from './components/TodoComponents/TodoForm';
+
+import './App.css';
+
+const taskList = [];
 
 class App extends React.Component {
-  // you will need a place to store your state in this component.
-  // design `App` to be the parent component of your application.
-  // this component is going to take care of state, and any change handlers you need to work with your state
-  render() {
-    return (
-      <div>
-        <h2>Welcome to your Todo App!</h2>
-      </div>
-    );
-  }
+	constructor() {
+		super();
+		this.state = {
+			todoList: taskList,
+			todoItem: '',
+			id: '',
+			date: '',
+			completed: ''
```

Comment:
Instead of declaring an empty array outside of App's state, you can declare the empty array on todoList itself